docs/design.md

```diff
@@ -633,6 +663,7 @@ A really good reference on tuning Linux installations with RocksDB is
 
 # Range Metadata
 
+[Peter: did you actually have numbers for the size of meta2 records?]
```

Comment:
Yes. Easy enough to recompute, though it was below 256 bytes and that upper bound still seems reasonable. I can take this section as well.